src/stream/tests/integration_tests/snapshot.rs

```diff
@@ -146,7 +147,7 @@ where
     for mut event in inputs {
         match &mut event {
             SnapshotEvent::Barrier(epoch) => {
-                tx.push_barrier(*epoch, false);
+                tx.push_barrier(test_epoch(*epoch), false);
```

Comment:
It is possible to change to epoch in `SnapshotEvent::Barrier` to a correctly shifted value?

Reply:
A bit hard, let's keep it like this for now.